### PR TITLE
Added copy key

### PR DESCRIPTION
**Pull request by sbailey-arm**
Signed-off-by: Samuel Bailey <samuel.bailey@arm.com>

### CODE REVIEW

File: protobuf/psa_copy_key.proto [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2019 Contributors to the Parsec project.
```
Comment (ionut-arm) [inline protobuf/psa_copy_key.proto:2]: 2020! 😄

**Comment by hug-dev** [comment]
Yes good point as it says:

> This function is primarily useful to copy a key from one location to another

but that does not apply to Parsec exactly as we don't use lifetime to specify locations.

I would say it's better to hold the copy operation until we have the design of something that would make sense in Parsec, like as you said, copying a key from one provider to another.

**Comment by ionut-arm** [comment]
@sbailey-arm - you ok with closing this PR for now, until we figure out how we want this to work for Parsec?

**Comment by sbailey-arm** [comment]
> 
> 
> @sbailey-arm - you ok with closing this PR for now, until we figure out how we want this to work for Parsec?

Yes, that's fine

**Pull request closed without merging**